services/terrain/src/terrain/clients/data_info.clj

```diff
@@ -286,8 +287,11 @@
    Returns:
      It returns a path-stat map containing an additional UUID field."
   [^String user ^UUID uuid]
-  (uuids/path-for-uuid user uuid))
-
+  (-> (raw/collect-stats user :ids [uuid])
+      :body
+      json/decode
+      (get-in ["ids" uuid])
+      walk/keywordize-keys))
```

Comment:
This code changes the return value very slightly, to no longer duplicate the object ID in both `:id` and `:uuid` keys. The latter is removed (as it was only previously added by `uuids/path-for-uuid`, and is not used by any of the callers of this method).